mautic/mautic
# Report - fix date data display for local timezone

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

In some cases could date field make display wrong data due UTC conversion. We fixed it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create custom field date for contact
3. Create some contacts  with values in that field
4. Create report with the list of contacts with the **date field** in the column of report
5. Just check If date field is displayed in report like before. Nothing is broken is a good enough test.

Just check if date column in report display correct date like before.
